components/suggestion/suggestion_list.d.ts

```diff
@@ -9,7 +9,7 @@ import React from 'react';
 // eslint-disable-next-line @typescript-eslint/no-empty-interface
 interface SuggestionItem {}
 
-interface Props {
```

Comment:
we will get rid of this file when we migrate `suggestion_list` so i would recommend defining the props in the date component